### DISCUSSION TITLE
Allow Typing Into Cells Once Highlighted

### DISCUSSION THREAD

**Discussion by MrChadMWood** [mathesar-foundation/mathesar#3233] [Feature Ideas]
In the current state, highlighting cells do not make them editable:

Non-Editable:
![image](https://github.com/centerofci/mathesar/assets/90083658/b756a948-e2ad-4b62-b352-8fc6d3ccb2d8)

This makes typing in data manually a little slower, as every cell requires double-click with the mouse.

### Feature Request:
Add the ability (or option) to type into a cell once it is highlighted. 
This way users can navigate across cells while entering data more quickly by -- for instance -- pressing tab, entering the data they want, pressing tab, entering the data they want, ...

**Comment by seancolsen** [comment]
I've converted this to an [issue](https://github.com/centerofci/mathesar/issues/3236) since it's a very specific and well-defined problem. We can continue the discussion there.
